applications/portal/frontend/src/components/viewers/detailsViewer/DetailsViewer.tsx

```diff
@@ -0,0 +1,483 @@
+                </Box>
+                <Box display="flex" alignItems="center" justifyContent="center" py={4}>
+                    <img src={pageImg} alt='' style={{ maxWidth: 500, maxHeight: 400 }} />
+                </Box>
+            </div>
+            <Box className={classes.pagination}>
+                <Typography>
+                    Page {page} of {data[tabIdx].files[selectedIndex].pages.length}
+                </Typography>
+                <Pagination
+                    count={data[tabIdx].files[selectedIndex].pages.length}
+                    size="large"
+                    page={page}
+                    variant="outlined"
+                    shape="rounded"
+                    onChange={handlePageChange}
+                />
+            </Box>
+            <Modal
+                open={openUploadDialog}
+                dialogActions={true}
+                actionText="Upload"
+                disableGutter={true}
+                handleClose={() => setOpenUploadDialog(false)}
+                handleAction={() => { }}
+                title={`Upload PDF to Population ${populationName}`}
+            >
+                <Box display="flex" alignItems="center" justifyContent="center" p={2}>
+                    <PdfFileDrop
+                        file={files[0]}
+                        setFile={(value: any) => handleAddFile(PDF_FILE, value)}
+                        hasErrors={validationErrors.has(PDF_FILE)}
+                    />
+                </Box>
+                <CategorySelect category={category} handleCategoryChange={handleCategoryChange} />
+                {
+                    files[0] !== undefined && <Box className={classes.addAnotherFileBox}>
+                        <AddPdfFileDrop
+                            file={files[1]}
+                            setFile={(value: any) => handleAddFile(PDF_FILE, value)}
+                            hasErrors={validationErrors.has(PDF_FILE)}
+                        />
+                    </Box>
+                }
+            </Modal>
+            <DeleteDialog
+                open={openDeleteDialog}
+                handleClose={() => setOpenDeleteDialog(false)}
+                actionText="Delete"
+                title="Delete this file?"
+                dialogActions={true}
+                handleAction={() => { }}
+            >
+                <Box className={classes.deleteModalBox}>
+                    <Typography>This action cannot be undone. Are you sure you want to delete “Electrophysiological properties of V2a
+                        interneurons in the lumbar spinal cord.pdf”?</Typography>
+                </Box>
+            </DeleteDialog>
+        </div>
+    ) : (
+        <div className={classes.container} style={{ alignItems: 'center', justifyContent: 'center' }}>
+            <p className={classes.noContentTitle}>Select a population to start viewing details</p>
+        </div>)
+};
+
+export default DetailsViewer
+
+const data = [
```

Comment:
Mocked data like this is ok but ideally, if you expect the component to have to fetch this data by itself you should mock the method responsible for the fetching instead of having a variable out of the scope of the component.